documentation/bibliography.bib

```diff
@@ -1710,6 +1721,15 @@ @book{Sterbenz1974
   title     = {Floating-point computation},
 }
 
+@book{Szebehely1967,
+  author    = {Szebehely, Victor G.},
+  publisher = {Academic Press},
+  date      = {1967},
+  isbn      = {978-0-12-395732-0},
+  subtitle  = {The restricted problem of three Bodies},
```

Comment:
The Restricted Problem of Three Bodies, see https://www.sciencedirect.com/science/article/pii/B9780123957320500015.